rpc.Dockerfile

```diff
@@ -10,6 +10,4 @@ ADD . /usr/src/rpc
 
 WORKDIR /usr/src/rpc
 RUN git config --global url."https://".insteadOf git://
-RUN yarn install
-
-CMD yarn rpc
\ No newline at end of file
+RUN yarn install
```

Comment:
The `rpc` service is built, but no command is running in the container, so it exits immediately.
```
docker ps
CONTAINER ID   IMAGE                   COMMAND                  CREATED         STATUS         PORTS     NAMES
f5f4aed490b1   etomic-swap-workspace   "docker-entrypoint.s…"   3 minutes ago   Up 3 minutes             etomic-swap-workspace-1
```

As hardhat uses its own network that seem to run in the same `workspace` container, the `rpc` service can be removed at all.

Reply:
Removed rpc service. Thanks!